modules/basics/pages/system-and-language-basics.adoc

```diff
@@ -65,6 +67,50 @@ To exit the GSQL shell, type either `exit` or  `quit` at the GSQL prompt:
 
 `GSQL> EXIT`  or `GSQL> QUIT`
 
+=== New token-based authentication method
+
+Starting with version *4.1.0*, GSQL now supports token-based authentication, allowing users to log into the GSQL shell using a token instead of a traditional username and password.
+
+*How to login using a token*
+
+To log into the GSQL shell using a token, you can use the `-t` or `--token` option, followed by the token value. The token can either be a plain-text token or a JWT token.
+[source,gsql]
+----
+gsql -t <token>
+----
+
+*Types of tokens*
+
+* *Plain-text Token:* A randomly generated string used for authentication. This is the older token format, still supported if not expired, but will eventually be deprecated.
+* *JWT Token:* A more secure and modern format for authentication. It consists of three parts: header, payload, and signature. JWT tokens are the default format for authentication starting from version 4.1.0.
+
+*Token login workflow*
+
+* *Login Process:* To log in using the `-t` option, simply provide a valid token with the `gsql -t <token>` command. If the token is valid, you will be authenticated and logged into the GSQL shell.
```

Comment:
A full example (login into the shell, run a few commands, exit the shell) is more useful than the comparison to secrets below.